src/application/clients/crux-gateway-bridge-config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "HOST_URL": {
+        "BROKER_HOST": "127.0.0.1",
+        "TCP_HOST": "127.0.0.1"
+    },
+    "PORTS": {
+        "BROKER_PORT": 1883,
+        "TCP_PORT": 4005,
+        "PUB_SUB_CLIENT_1_PORT": 3000,
```

Comment:
remove every single attribute you are not using in code right now

Reply:
done